.github/workflows/release.yaml

```diff
@@ -25,25 +25,18 @@ jobs:
       - name: copy release workload assets into _dist
         run: |
           mkdir -p _dist
-          cp ./deployments/workloads/spin/runtime.yaml _dist/spin_runtime.yaml
-          cp ./deployments/workloads/spin/workload.yaml _dist/spin_workload.yaml
-          cp ./deployments/workloads/slight/runtime.yaml _dist/slight_runtime.yaml
-          cp ./deployments/workloads/slight/workload.yaml _dist/slight_workload.yaml
-      - uses: actions/download-artifact@v3
```

Comment:
We need the release artifacts downloaded still. Can you add this back in?

Reply:
oops you're right! That was a mistake. I will add them back